### PR TITLE
Add dependencies for Fedora

### PR DESCRIPTION
**Pull request by am97**

### CODE REVIEW

File: support/doc/dependencies.md [hunk 1]
```diff
@@ -84,6 +84,74 @@ $ sudo service redis start
...
+
+_from [PostgreSQL documentation](https://www.postgresql.org/download/linux/redhat/):_
+> Due to policies for Red Hat family distributions, the PostgreSQL installation will not be enabled for automatic start or have the database initialized automatically.
+```
+# PostgreSQL
+sudo postgresql-setup initdb
+sudo systemctl enable postgresql.service
+sudo systemctl start postgresql.service
+# Nginx
+sudo systemctl enable nginx.service
```
Comment (BO41) [inline support/doc/dependencies.md]: does nginx really need a started service on fedora?
on other distros it seems not needed

Comment (am97) [reply support/doc/dependencies.md]: It is not started by default, I guess it is some security policy

Comment (am97) [reply support/doc/dependencies.md]: According to [fedora nginx guide](https://fedoraproject.org/wiki/Nginx) you should start it after installing

Comment (BO41) [reply support/doc/dependencies.md]: okay, good :+1:

**Comment by rigelk** [comment]
Thanks @am97, @BO41 (and @daftaupe for his quick look from IRC!)

**Pull request merged**